### PR TITLE
BAEL-3941

### PR DESCRIPTION
**Pull request by vatsalgosar**
- Code snippet for preserving line breaks while using Jsoup to parse HTML documents

### CODE REVIEW

File: jsoup/src/test/java/com/baeldung/jsoup/PreservingLineBreaksUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+public class PreservingLineBreaksUnitTest {
+
+    @Test
+    public void whenBackSlashNNewLineCharacter_thenPreserveLineBreak() {
+        String strHTML = "<html><body>Hello\nworld</body></html>";
+        Document.OutputSettings outputSettings = new Document.OutputSettings();
+        outputSettings.prettyPrint(false);
+        String strWithNewLines = Jsoup.clean(strHTML, "", Whitelist.none(), outputSettings);
+        assertEquals(strWithNewLines, "Hello\nworld");
```
Comment (KevinGilmore) [inline jsoup/src/test/java/com/baeldung/jsoup/PreservingLineBreaksUnitTest.java:17]: swap order of arguments

File: jsoup/src/test/java/com/baeldung/jsoup/PreservingLineBreaksUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        Document jsoupDoc = Jsoup.parse(strHTML);
+        Document.OutputSettings outputSettings = new Document.OutputSettings();
+        outputSettings.prettyPrint(false);
+        jsoupDoc.outputSettings(outputSettings);
+        jsoupDoc.select("br").before("\\n");
+        jsoupDoc.select("p").before("\\n");
+        String str = jsoupDoc.html().replaceAll("\\\\n", "\n");
+        String strWithNewLines =
+                Jsoup.clean(str, "", Whitelist.none(), outputSettings);
+        assertEquals(strWithNewLines, "Hello\nWorld\nParagraph");
```
Comment (KevinGilmore) [inline jsoup/src/test/java/com/baeldung/jsoup/PreservingLineBreaksUnitTest.java:37]: swap order of arguments

File: jsoup/src/test/java/com/baeldung/jsoup/PreservingLineBreaksUnitTest.java (hunk 1 above)
Comment (vatsalgosar) [reply jsoup/src/test/java/com/baeldung/jsoup/PreservingLineBreaksUnitTest.java]: Done.

File: jsoup/src/test/java/com/baeldung/jsoup/PreservingLineBreaksUnitTest.java (hunk 2 above)
Comment (vatsalgosar) [reply jsoup/src/test/java/com/baeldung/jsoup/PreservingLineBreaksUnitTest.java]: Done.

**Pull request merged**